src/js/core.js

```diff
@@ -395,6 +395,9 @@ function MediumEditor(elements, options) {
         AnchorPreview: AnchorPreview
     };
 
+    MediumEditor.Util = Util;
+    MediumEditor.Selection = Selection;
```

Comment:
Meh, we can just worry about it later.  I'd rather get your changes in then worry too much about semver right now.